### ISSUE TITLE
Webcam helper is not working

### ISSUE THREAD

**Issue by MichalSuchy** [OSadovy/webcam-helper-nvda-addon#10]
I installed Webcam helper to NVDA 2026. 1 and it does not work. I'm sending the log.
Error importing global plugin webcamHelper
Traceback (most recent call last):
  File "globalPluginHandler.pyc", line 23, in listPlugins
  File "importlib\__init__.pyc", line 88, in import_module
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1023, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "C:\Users\suchy\AppData\Roaming\nvda\addons\webcamHelper\globalPlugins\webcamHelper\__init__.py", line 11, in <module>
    import numpy as np
  File "C:\Users\suchy\AppData\Roaming\nvda\addons\webcamHelper\globalPlugins\webcamHelper\3rdparty\numpy\__init__.py", line 148, in <module>
    from . import _distributor_init
  File "C:\Users\suchy\AppData\Roaming\nvda\addons\webcamHelper\globalPlugins\webcamHelper\3rdparty\numpy\_distributor_init.py", line 26, in <module>
    WinDLL(os.path.abspath(filename))
    ~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "ctypes\__init__.pyc", line 361, in __init__
  File "ctypes\__init__.pyc", line 379, in _load_library
OSError: [WinError 193] %1 not a valid Win32 application
